packages/web-components/src/tree-item/tree-item.template.ts

```diff
@@ -0,0 +1,72 @@
+import { children, elements, html, when } from '@microsoft/fast-element';
+import { FluentDesignSystem } from '../fluent-design-system.js';
+import type { TreeItem } from './tree-item.js';
+import { treeItemLevelToken } from './tree-item.style.js';
+
+// We don't put the icon into the icons/index.ts file because
+// this icon is the default chevron icon of the tree-item component,
+// it would be contributed to the upstream later.
+const chevronIcon = html`
+  <svg width="12" height="12" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
+    <path
+      d="M4.65 2.15a.5.5 0 000 .7L7.79 6 4.65 9.15a.5.5 0 10.7.7l3.5-3.5a.5.5 0 000-.7l-3.5-3.5a.5.5 0 00-.7 0z"
+    ></path>
+  </svg>
+`;
+
+export const template = html<TreeItem>`
+  <template
+    tabindex="-1"
+    slot="${x => (x.isNestedItem ? 'item' : void 0)}"
+    class="${x => x.calculatedClassName}"
+    @focusin="${(x, c) => x.handleFocus(c.event as FocusEvent)}"
+    @focusout="${(x, c) => x.handleBlur(c.event as FocusEvent)}"
+    style="${x => `${treeItemLevelToken}: ${x.depth};`}"
+    ${children({
+      property: 'childTreeItems',
+      filter: elements(`${FluentDesignSystem.prefix}-tree-item`),
+    })}
+  >
+    <div class="positioning-region" part="positioning-region">
+      <div class="content-region" part="content-region">
+        <span class="selection-region" part="selection-region"></span>
+        <span
```

Comment:
If the chevron is just a visual indicator, this `<span>` should have `aria-hidden="true"` to hide it from screen readers (and screen readers can get the state from `aria-expanded` maybe).

Reply:
right now the fluent-tree-item is served as clickable area for expanding, we should also support chevron as clickable area, in which case user can click chevron icon to expand/collapse instead of whole tree item, and let screen reader to get correct area.

Reply:
Interactive elements should not be nested, and screen readers likely won't announce the nested ones. Nest interactive elements can cause confusion particularly for vision impaired users.